### PR TITLE
Buyer/Seller logbook

### PR DESCRIPTION
**Pull request by DanielmAaron**
Tasks 
Order.js route
- [x] Assigns user to logbook based on role
- [x] Add bid/offer logic
- [x] Filter logic

Buyerlogbook
- [x] Create buyer logbook Dashboard
- [x] Add bids
- [x] View Bids
- [x] Apply filter to bids

Sellerlogbook
- [x] Create buyer logbook Dashboard
- [x] Add offers
- [x] View offers
- [x] Apply filter to offers

### CODE REVIEW

**Comment by DanielmAaron** [comment]
Tested
Interfaces:
Buyerlogbook dashboard
Screenshot 2022-05-10 at 15 09 14
Buyerlogbook add bid
Screenshot 2022-05-10 at 15 09 20
Buyerlogbook view bid
Screenshot 2022-05-10 at 15 09 27

sellerlogbook dashboard
Screenshot 2022-05-10 at 15 03 52
sellerlogbook add offer
Screenshot 2022-05-10 at 15 08 41
sellerlogbook view offer
Screenshot 2022-05-10 at 15 08 50

**Comment by DanielmAaron** [comment]
Added filter to buyer and seller logbook
Screenshot 2022-05-11 at 13 06 33
Screenshot 2022-05-11 at 13 07 10

File: routes/buyer.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    }
+  }
+);
+
+//GET Filtered buyer dashboard
+router.get(
+  '/filter/produce/:range',
+  require('connect-ensure-login').ensureLoggedIn({ redirectTo: '/app/auth/login' }),
+  function (req, res, next) {
+    if (req.user.role === ROLES.Buyer) {
```
Comment (jajukajulz) [inline routes/buyer.js:71]: let's add an admins and superusers here i.e. in addition to buyers

File: routes/seller.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+var sequelise = require('../config/db/db_sequelise');
+
+var models = initModels(sequelise);
+
+//GET seller dashboards
+router.get(
+  '/',
+  require('connect-ensure-login').ensureLoggedIn({ redirectTo: '/app/auth/login' }),
+  function (req, res, next) {
+    if (req.user.role === ROLES.Seller) {
```
Comment (jajukajulz) [inline routes/seller.js:27]: I will merge this shortly but let's add an admins and superusers here i.e. in addition to sellers

File: routes/seller.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    }
+  }
+);
+
+//GET Filtered seller dashboard
+router.get(
+  '/filter/produce/:range',
+  require('connect-ensure-login').ensureLoggedIn({ redirectTo: '/app/auth/login' }),
+  function (req, res, next) {
+    if (req.user.role === ROLES.Seller) {
```
Comment (jajukajulz) [inline routes/seller.js:71]: let's add an admins and superusers here i.e. in addition to sellers

**Pull request merged**